CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -8,7 +8,9 @@ Previous change logs can be found at [CHANGELOG-3.5](https://github.com/etcd-io/
 
 <hr>
 
-## v3.6.0-rc.4 (TBC)
+## v3.6.0-rc.5 (TBC)
```

Comment:
As mentioned on release call I don't think we will need an rc5 release hopefully. It's ok to leave it in I think and we will remove it in future if we decide to publish `v3.6.0` without an rc5.

Reply:
OK. Removed it